pkg/bloomfilter/filter.go

```diff
@@ -0,0 +1,85 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package bloomfilter
+
+import (
+	"encoding/binary"
+	"math"
+
+	"storj.io/storj/pkg/storj"
+)
+
+// Filter is a bloom filter implementation
+type Filter struct {
+	seed           int
+	k              int
+	bitsPerElement int
+	table          []byte
+}
+
+// NewFilter returns a new custom filter
+func NewFilter(expectedElements int, p float64) *Filter {
+	var f Filter
+	f.seed = 8 // TODO allow another seed value - have to handle the reach of the end of the array
+	f.bitsPerElement = int(-1.44*math.Log2(p)) + 1
+	f.k = int(float64(f.bitsPerElement)*math.Log(2)) + 1
+	m := (expectedElements * f.bitsPerElement) // total number of bits in the array
+	f.table = make([]byte, m/8)
+	return &f
+}
+
+// Add adds an element to the bloom filter
+func (cf *Filter) Add(pieceID storj.PieceID) {
```

Comment:
I didn't see it yesterday, but `cf` isn't a good name.
Following our [current conventions]https://github.com/storj/docs/blob/master/code/Style.md#variable-naming) my guess is that it has to be `filter`